packages/lab/stories/pill-next/pill-next.qa.stories.tsx

```diff
@@ -14,20 +14,18 @@ export const ExamplesGrid: StoryFn<QAContainerProps> = (props) => {
   const { className, ...rest } = props;
   return (
     <QAContainer cols={3} height={300} itemPadding={3} width={1300} {...rest}>
+      <PillNext className={className}>Default Pill</PillNext>
       <PillNext className={className} onClick={noop}>
-        Default Pill
+        Clickable Pill
       </PillNext>
       <PillNext className={className} onClick={noop} disabled>
         Disabled Pill
       </PillNext>
-      <PillNext className={className} icon={<FavoriteIcon />} onClick={noop}>
-        With Icon Pill
-      </PillNext>
-      <PillNext onClose={noop} className={className}>
-        Closable Pill
+      <PillNext className={className} onClick={noop}>
+        <FavoriteIcon /> With Icon Pill
       </PillNext>
       <PillNext onClick={noop} className={className}>
-        Extra extra long Pill label example.
+        <span>Extra extra long Pill label example.</span>
```

Comment:
I'm not that sure about using `.saltPillNext > *` to control whether things get truncated or not. Feels like we're prescribing something very specific